### ISSUE TITLE
this mod isnt friendly for people with low iq

### ISSUE THREAD

**Issue by Pinecone6442** [DonBruce64/MinecraftTransportSimulator#1340]
TOO HARD

**Comment by boot2big** [comment]
There are a plethora of tutorials and guides on how to use this mod and all of its little features, and I've even considered making a constantly updating and community contributable guide on the little quirks and tricks one can use that aren't necessarily intended features, but aren't outright exploits.

The amount of help pages, instructional videos, previous community comments on CurseForge, YouTube and Discord as well as the _two ingame manuals you received upon loading into the game_ are more than enough for anyone with a "low iq" to work through.

You ain't a low IQ (necessarily), you lazy!
In conclusion,

![y5bl7sml0x](https://user-images.githubusercontent.com/53581325/192135340-b5104901-1a7b-4021-beac-bead1136b9d2.gif)


P.S: The main developer and various pack authors have already been implementing this suggestion in one way or another _anyways,_ so this ticket is not only invalid but technically a duplicate of various others!

**Comment by LauraDarkez** [comment]
B2B is correct on this comment. i expect this ticket to be closed promptly upon don becoming active
